templates/services/merge/validator/lodestar.tmpl

```diff
@@ -1,39 +1,10 @@
 {{/* lodestar.tmpl */}}
 {{ define "validator" }}
-  validator-import:
-    container_name: validator-import-client
-    image: ${VL_IMAGE_VERSION}{{with .VlCustomCfg}}
-    depends_on:
-      config_consensus:
-        condition: service_completed_successfully{{end}}
-    networks:
-      - sedge
-    volumes:
-      - ${KEYSTORE_DIR}:/keystore
-      - ${VL_DATA_DIR}:/data{{if .VlCustomCfg}}{{if .VlRemoteCfg}}
-      - ./config.yml:/network_config/config.yaml{{end}}{{if .VlRemoteGen}}
-      - ./genesis.ssz:/network_config/genesis.ssz{{end}}{{if .VlRemoteDpl}}
-      - ./deploy_block.txt:/network_config/deploy_block.txt{{end}}{{end}}
```

Comment:
I don't see these volumes being set up in `setupValidatorImportContainer`

Reply:
Please check this commit https://github.com/NethermindEth/sedge/pull/208/commits/09743fd0d382943870ce2066823bbcdded15964a

Reply:
Sorry, I still don't see you are mounting the validator data directory. Have you tried a full setup and has it worked for you?

Reply:
Here the idea is the same that I explain [here](https://github.com/NethermindEth/sedge/pull/208#discussion_r1111677604), but in this case, the lines are [these](https://github.com/NethermindEth/sedge/blob/a981c5d0edc8fab4c00132be5f3a111295716181/cli/actions/importKeys.go#L222-L225)